content/37c3_potluckctf_2023/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: 37c3 Potluckctf 2023
+date: 2023-12-29
+description: Writeups for [37c3 Potluckctf 2023]
+
```

Comment:
No final place ?

Reply:
no, I actually don't know =)) I just joined some guys playing. I don't even know the team name

Reply:
ookey